plugins/modules/aci_l3out.py

```diff
@@ -46,20 +51,84 @@
   route_control:
     description:
     - Route Control enforcement direction. The only allowed values are export or import,export.
+    - The APIC defaults to C(export) when unset during creation.
     type: list
     elements: str
     choices: [ export, import ]
     aliases: [ route_control_enforcement ]
+  mpls:
+    description:
+    - Indicate whether MPLS is enabled or not.
+    - The APIC defaults to C(no) when unset during creation.
+    type: str
+    choices: [ "no", "yes" ]
   l3protocol:
     description:
-    - Routing protocol for the L3Out
+    - Routing protocol for the L3Out.
+    - Protocols must be re-provided with their paramaters each time an l3out with existing protocols is modified if the protocols were to be kept.
```

Comment:
Suggestion -
- Protocols already associated with an l3out must be provided again when the l3out is modified if the associated protocols are to be kept.